api/src/main/java/org/pmiops/workbench/mailchimp/MailChimpServiceImpl.java

```diff
@@ -65,8 +68,12 @@ public String getMember(String contactEmail) {
 
     try(MailchimpClient client = new MailchimpClient(apiKey)) {
       return client.execute(method);
-    } catch (IOException | MailchimpException e) {
-      throw new RuntimeException(e);
+    } catch (IOException e) {
+      WorkbenchException we = new WorkbenchException(new ErrorResponse().statusCode(500).errorClassName("IO Exception").message(e.getMessage()));
```

Comment:
The error class name could be less string-y with `e.getClass().getName()` and then be the same for both classes. I would love to simplify this and still be able to get the mailchimp error status, but I don't think that's possible.